autopush/db.py

```diff
@@ -466,6 +486,23 @@ def __init__(self, **kwargs):
     def __getattr__(self, name):
         return getattr(self._resource, name)
 
+    def get_latest_message_tablename(self, prefix="message"):
+        # type: (Optional[str]) -> str  # noqa
+        """Fetches the name of the last message table"""
+        if not self._latest_message_table:
+            client = self._resource.meta.client
+            paginator = client.get_paginator("list_tables")
+            tables = []
+            for table in paginator.paginate().search(
+                    "TableNames[?contains(@,'{}')==`true`]|sort(@)[-1]".format(
```

Comment:
Contains? Is this going to require it as the prefix or will it find the prefix anywhere in the table name?

Reply:
Right now our table prpefix's look like this "{dev/stage/prod}.autopush.message", so it should work ok for us as Contains, but it'd prolly be safer if it required the table to start with the prefix.